app/Check/Commands/SlackCheckStatusesCommand.php

```diff
@@ -131,6 +139,14 @@ protected function execute(
 			);
 
 			$this->slackNotifier->notify('#monitoring', $message, $color);
+
+			foreach ($check->project->userSlackNotifications as $user) {
+				$identity = $this->userRepository->getById($user->user->id);
```

Comment:
Domnívám se, že by tam mělo stačit `$identity = $user->user;`

Reply:
> Domnívám se, že by tam mělo stačit $identity = $user->user;

Ta metoda požaduje přímo ID